packages/vue/src/Form/DatePicker/DatePicker.vue

```diff
@@ -111,6 +115,25 @@ const resetCalendar = () => {
 };
 
 const defaultDateRange = () => [dayjs().toDate(), dayjs().toDate()];
+
+const disableAllDates = (value) => {
+  const date = dayjs(value);
+  const isInCurrentMonth = (date) => date.get('month') === dayjs().get('month');
+  return isInCurrentMonth(date);
+}
+
+const isCalendarIndefinite = ref(false);
+
+const handleIndefinite = (event) => {
+  console.log('event from calendar', event)
+  if (event) {
+    // emit("update:modelValue", 'Indefinite');
+    isCalendarIndefinite.value = true;
+  } else {
+    isCalendarIndefinite.value = false;
```

Comment:
```suggestion
    isCalendarIndefinite.value = false;
```

Reply:
And remove `if (event) {
    // emit("update:modelValue", 'Indefinite');
    isCalendarIndefinite.value = true;
  } else {
    isCalendarIndefinite.value = false;
  }`